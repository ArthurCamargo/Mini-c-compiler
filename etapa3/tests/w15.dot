digraph G {
n0;
n0 -> n1;
n1;
n1 [label="break"];
n0 [label="f"];

}