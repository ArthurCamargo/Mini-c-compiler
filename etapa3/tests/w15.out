0x56125278a350
0x56125278a350, 0x56125278a3c0
0x56125278a3c0
0x56125278a3c0 [label="break"];
0x56125278a350 [label="f"];
